migration/migration_test.go

```diff
@@ -38,3 +38,24 @@ func TestConcurrentMigrations(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestMigrateWorkItemDescriptions(t *testing.T) {
```

Comment:
To me it seems like this test function is testing that migrateWorkItemDescriptionsToMarkup executes and returns without errors. But it doesn't check the result (e.g. was the markup set to a correct value?).